examples/kitchensink/package.json

```diff
@@ -1,23 +1,25 @@
 {
-  "name": "frint-kitchensink-app-counter",
+  "name": "frint-kitchensink",
   "version": "0.0.1",
-  "description": "Kitchensink counter app in Frint",
+  "description": "Kitchensink app in Frint",
   "main": "index.js",
   "scripts": {
-    "build": "webpack",
-    "test": "mocha --opts ./test/config/mocha.opts ./test/**/*.spec.js"
+    "build": "webpack"
   },
   "author": "Travix International B.V.",
   "license": "MIT",
   "dependencies": {
     "frint": "^1.0.1",
-    "frint-model": "^1.0.1",
-    "frint-react": "^1.0.1",
-    "frint-store": "^1.0.0",
+    "frint-model": "^1.3.0",
+    "frint-react": "^1.3.0",
+    "frint-store": "^1.3.0",
     "frint-test-utils": "^1.0.0",
     "react": "^15.4.2",
     "react-addons-test-utils": "^15.4.2",
-    "react-dom": "^15.4.2"
+    "react-dom": "^15.4.2",
+    "http-server": "^0.9.0",
+    "frint-preset-travix": "^1.3.0",
```

Comment:
if you are already installing the packages on demand (like `frint-model`, `frint-react`), you don't need to install the preset any more.